### PR TITLE
added days function to the script

### PR DESCRIPTION
**Pull request by rajpalc7**

### CODE REVIEW

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
 /**
  * uptime-rc-webhook.js
  * Add Uptime.com notifications via WebHook in Rocket.Chat
- * Usage on uptime.com webhook integration setup: put this in the Custom HTTP Headers to tag ppl in the message`contactusers: @<user1_rocketchat_account> @<user2_rocketchat_account> ... ,`
+ * Usage on uptime.com webhook integration setup: put this in the Custom HTTP Headers to tag ppl in the messagecontactusers: @<user1_rocketchat_account> @<user2_rocketchat_account> ... ,
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js]: please do not remove the backward quote, it's meant to be an example

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
  */
 
-/* globals console, _, s */
+/ globals console, _, s /
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js]: `/* */` is used for comments, does it still work without?

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+  const totalMinutes = Math.floor(totalSeconds / 60);
+  const minutes = totalMinutes % 60;
+  const totalHours = Math.floor(totalMinutes / 60);
+  const hours = totalHours % 24;
+  const days = Math.floor(totalHours / 24);
+
+  return `${days}d:${hours}h:${minutes}m:${seconds}s`;
+}
+
+/ exported Script /
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js]: same here!

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 4]
```diff
@@ -46,16 +53,16 @@ class Script {
...
+	  title: titleText,
+	  title_link: titleLink,
+	  text: attachmentText,
+	  color: attachmentColor
+	}]
       }
     };
   }
-}
+}
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js]: please add the empty line at the end of the file.

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 5]
```diff
@@ -46,16 +53,16 @@ class Script {
 
     return {
       content:{
-        alias: USERNAME,
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js:49]: Looks like you are switching it from spaces to tabs, it's generally better to use space so the indentation don't change in different places. Please revert back!

**Review by ShellyXueHan — CHANGES_REQUESTED** [review]
Thank you for the contribution! Please take a look at the comments :)

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 6]
```diff
@@ -8,12 +8,23 @@
 const USERNAME = 'Uptime.com Alerts';
 const AVATAR_URL = 'https://avatars.githubusercontent.com/u/54849620?s=200&v=4';
 
+  const time = (totalSeconds) => {
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js]:
```suggestion
const time = (totalSeconds) => {
```

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 7]
```diff
@@ -8,12 +8,23 @@
...
+  const totalHours = Math.floor(totalMinutes / 60);
+  const hours = totalHours % 24;
+  const days = Math.floor(totalHours / 24);
+
+  return `${days}d:${hours}h:${minutes}m:${seconds}s`;
+}
+
 /* exported Script */
-class Script {
+  class Script {
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js]:
```suggestion
class Script {
```

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 8]
```diff
@@ -8,12 +8,23 @@
...
   /**
    * @params {object} request
    */
-  process_incoming_request({ request }) {
+    process_incoming_request({ request }) {
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js]:
```suggestion
  process_incoming_request({ request }) {
```

File: uptime/monitor-setup/uptime-rc-webhook.js [hunk 9]
```diff
@@ -22,20 +33,16 @@ class Script {
     let statusText = `DOWN`;
     let isUp = data.alert.is_up;
     if (isUp) {
-      attachmentColor = `#36A64F`;
-      statusText = `UP`;
+     attachmentColor = `#36A64F`;
+     statusText = `UP`;
```
Comment (ShellyXueHan) [inline uptime/monitor-setup/uptime-rc-webhook.js]:
```suggestion
      attachmentColor = `#36A64F`;
      statusText = `UP`;
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
You still have some whitespace issues to address @rajpalc7.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
That looks way better @rajpalc7.  Please squash and rebase the changes into a single commit on the latest code.  Thanks

**Review by ShellyXueHan — APPROVED** [review]
LGTM, thank you!

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**